libraries/codegen-tooling/inc/can_msg_defs.h

```diff
@@ -24,7 +24,6 @@ typedef enum {
 typedef enum {
   SYSTEM_CAN_MESSAGE_BPS_HEARTBEAT = 0,
   SYSTEM_CAN_MESSAGE_SET_RELAY_STATES = 1,
-  SYSTEM_CAN_MESSAGE_GET_RELAY_STATES = 2,
```

Comment:
Purposely got rid of "GET RELAY STATES"?

Reply:
arshan removed it in the video remember?

Reply:
Ah you merged with #54 ic.